### ISSUE TITLE
OpenVZ issue: Invalid Argument

### ISSUE THREAD

**Issue by hank** [cloudflare/boringtun#72]
I've got Ubuntu 16.04 running in an OpenVZ VPS instance, kernel:

```bash
Linux 2.6.32-042stab136.1 #1 SMP Wed Feb 27 09:04:24 MSK 2019 x86_64 x86_64 x86_64 GNU/Linux
```
TUN is enabled:

```bash
# cat /dev/net/tun
cat: /dev/net/tun: File descriptor in bad state
```

Here's what I get:

```bash
# boringtun -f --disable-multi-queue --disable-drop-priviliges wgnet0
Timer("Invalid argument")
# boringtun -f --disable-multi-queue wgnet0
Timer("Invalid argument")
# boringtun -f wgnet0
Timer("Invalid argument")
```
I've done this already:
```bash
# setcap cap_net_admin+epi boringtun
```
And it didn't throw any errors. Here's the current state:

```bash
# getcap ~/.cargo/bin/boringtun
/root/.cargo/bin/boringtun = cap_net_admin+eip
```

Any advice appreciated!

**Comment by vkrasnov** [comment]
It looks like a duplicate of #51

**Comment by vkrasnov** [comment]
I opened a PR to fix this and #51

**Comment by hank** [comment]
This totally worked. Thanks!
